visits/urls.py

```diff
@@ -7,4 +7,6 @@
     path('doctors', views.doctors_list, name="doctors"),
     path('available/<int:id>/reservation', views.reservation, name="reservation"),
     path('available', views.calendar, name="calendar"),
+    path('schedule/<int:id>', views.schedule_details, name="schedule_details"),
```

Comment:
I would suggest to change endpoints' names. What is the difference between _available/<int:id>_ and _schedule/<int:id>_  ?